frontend/lib/util/hook/useNumericParam.ts

```diff
@@ -0,0 +1,23 @@
+import { useParams } from "react-router-dom";
+
+import { parseIntStrict } from "../strings";
+
+// Takes a parameter key, then tries to find it in the URL params.
+// When it is found, it tries to validate it as a number and returns it.
+// Throws an error when the paramater key is not found or not an integer
+export function useNumericParam(parameterKey: string): number {
+  const params = useParams();
+  const param = params[parameterKey];
+
+  if (!param) {
+    throw Error(`Parameter ${parameterKey} does not exist`);
+  }
+
+  // Matches only if the whole string is numeric
+  const parsedParam = parseIntStrict(param);
+  if (!parsedParam) {
```

Comment:
Technically this doesn't work when the integer is `0` right now. Shouldn't be a problem for all the id params we have right now, given they start at `1`, but for other numeric values it might be an issue.

Reply:
Maybe this then?
```suggestion
  if (parsedParam !== undefined) {
```

Reply:
[8e29b14](https://github.com/kiesraad/abacus/pull/193/commits/8e29b148112e110059702cb33b1cd246bc8dfedf) addresses this